galaxykit/collections.py

```diff
@@ -12,6 +12,8 @@
 from orionutils.generator import build_collection
 from .client import GalaxyClientError
 
+from . import namespaces
```

Comment:
It's being added in this PR .. but not being used .. I don't think anyone is using it ;)

Reply:
But we are talking about version, not repository. Version was there before this PR and if we switch its order with repository, if anyone uses version, it will stop working.